app/src/debug/java/com/artemzin/qualitymatters/ui/fragments/DeveloperSettingsFragment.java

```diff
@@ -105,6 +108,12 @@ void onHttpLoggingLevelChanged(int position) {
         presenter.changeHttpLoggingLevel(((HttpLoggingLevel) httpLoggingLevelSpinner.getItemAtPosition(position)).loggingLevel);
     }
 
+    @OnClick(R.id.developer_settings_restart_app)
+    void onRestartAppClick() {
+        final FragmentActivity activity = getActivity();
+        ProcessPhoenix.triggerRebirth(activity, new Intent(activity, activity.getClass()));
```

Comment:
@lenguyenthanh, I considered this, but
• current presenter knows nothing about Android SDK-dependent types (context, view etc).
• I followed existing approach - to work  with context within this fragment like it's done e.g. in [`showLog()`](https://github.com/artem-zinnatullin/qualitymatters/blob/d101253d7e4ad58ea2bd03bde953e29700253fd4/app/src/debug/java/com/artemzin/qualitymatters/ui/fragments/DeveloperSettingsFragment.java#L210)